module1/homework/grayscaleImages/main.cpp

```diff
@@ -40,11 +41,19 @@ std::array<std::array<uint8_t, 32>, 32> generateNinja() {
     };
 }
 
+void printCompressed(const std::vector<std::pair<uint8_t, uint8_t>>& map) {
+    for (auto [color, count] : map) {
+        std::cout << "{" << static_cast<int>(color) << ", " << static_cast<int>(count) << "}, ";
+    }
+    std::cout << "\n";
```

Comment:
interesting. Why you implement this?

Reply:
I was checking correctness of my solution with this function and I forgot to remove it after the job was done! My bad